code/go/0chain.net/core/sortedmap/sortedmap.go

```diff
@@ -0,0 +1,92 @@
+package sortedmap
+
+import (
+	"sort"
+
+	"golang.org/x/exp/constraints"
+)
+
+// Map is a map that provides methods to return sorted values or keys
+type Map[K constraints.Ordered, V any] struct {
```

Comment:
we used 	"github.com/emirpasic/gods/maps/treemap"
in the blobber, maybe we can use it here

Reply:
The sortedmap is a wrapper around standard map that provide method or function to get sorted values. It's mainly for DRY code like [this](https://github.com/0chain/0chain/blob/58585100ff45cfcc003041260ad6d6f1243c951c/code/go/0chain.net/smartcontract/stakepool/stakepool.go#L201-L215). Basically we can't use treemap to store data in MPT due to the msgp. So add this package to sort existing map can help to avoid repeated code.